### DISCUSSION TITLE
Is there a way to configure several endpoints, corresponding to different schema files?

### DISCUSSION THREAD

**Discussion by frans-a** [Netflix/dgs-framework#288] [Q&A] [answered]
My use case is a test application which mocks several different GraphQL endpoints from existing external services.

Is it then possible with DGS to serve different schemas through several endpoints, within the same server application, instead of one schema and the default /graphql?

**Comment by srinivasankavitha** [comment, marked as answer]
No, we don't have a way to configure this setup. You can change the location from which the schema file is loaded (defaults to src/main/resources/schema), but cannot configure different endpoints out of the box.

**Reply by frans-a** [reply]
Thank you for this answer.
